go/protocol/keybase1/teams.go

```diff
@@ -1794,6 +1794,10 @@ type TeamOperation struct {
 	SetMemberShowcase      bool `codec:"setMemberShowcase" json:"setMemberShowcase"`
 	ChangeOpenTeam         bool `codec:"changeOpenTeam" json:"changeOpenTeam"`
 	LeaveTeam              bool `codec:"leaveTeam" json:"leaveTeam"`
+	JoinTeam               bool `codec:"joinTeam" json:"joinTeam"`
+	SetPublicityAny        bool `codec:"setPublicityAny" json:"setPublicityAny"`
+	ListFirst              bool `codec:"listFirst" json:"listFirst"`
+	ShowcaseSettings       bool `codec:"showcaseSettings" json:"showcaseSettings"`
```

Comment:
Is it the same as `SetTeamShowcase        bool `codec:"setTeamShowcase" json:"setTeamShowcase"`` above?

Reply:
There are 2 different showcase settings, one to showcase on your profile, one to showcase on your own profile (ShowcaseSettings) and one to allow members to do so (SetMemberShowcase).

Reply:
@zanderz But now there are four showcase settings in TeamOperation -- SetTeamShowcase, SetMemberShowcase, SetPublicityAny, ShowcaseSettings.  @zapu is asking whether ShowcaseSettings is identical to the one of the other three that already exist.

Reply:
I think we only need 3 of them:
1) `SetTeamShowcase` - can current user set team to be promoted in popular-teams
2) `SetMemberShowcase` - can current user promote this team in their profile
3) `SetPublicityAny` - can current user flip the "any member can showcase in their profile" switch (this switch is checked in 2)

Also 3 has the same rules as 1 (implicit admin or explicit admin/owner), hence why it was missing when this function was first made. Let me know if I'm wrong in any of that

Reply:
ah, ok: I was going by the existing code around line 350 of shared/teams/team/index.js, which hides the showcase settings sometimes, which I replaced with ShowcaseSettings. But @cecileboucheron tells me that is wrong and it should always be shown, but disabled sometimes. Will remove.